src/Modal/Modal.js

```diff
@@ -105,13 +96,18 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
       <Portal>
         <Transition in={isOpen} timeout={animationDuration}>
           {state => (
-            <Backdrop transitionState={state} onClick={handleBackdropClick}>
-              <FocusLock lockProps={{ style: { maxHeight: '100%' } }} disabled={!isOpen}>
-                <ModalContent transitionState={state} onClick={handleContentClick} {...props}>
+            <Backdrop transitionState={state}>
+              <FocusOn
```

Comment:
What prompted the new library? Can we remove the `react-focus-lock` library so we don't have redundant libs doing similar things?

Reply:
👍 I believe the main goal was accessibility. `react-focus-on` creates an accessible experience. So, now the modal and dropdown should be more accessible. I went ahead and removed the `react-focus-lock` library, and am now using `react-focus-lock` exclusively. (Which is maintained by the same group, and also includes `react-focus-lock` in it's dependencies.)